package.json

```diff
@@ -7,25 +7,31 @@
     "bugs": "https://github.com/privacy-scaling-explorations/zk-kit/issues",
     "private": true,
     "scripts": {
-        "build": "yarn build:libraries && yarn compile:contracts",
-        "build:libraries": "yarn workspaces foreach -A -t --no-private run build",
-        "compile:contracts": "yarn workspaces foreach -A run compile",
-        "test": "yarn test:libraries && yarn test:contracts && yarn test:circuits",
-        "test:libraries": "jest --coverage",
+        "build": "turbo build",
+        "_build:libraries": "yarn workspaces foreach -A -t --no-private run build",
```

Comment:
`_script`(s) added for convenience so they can just be ran like before with `yarn [script]` and still benefit from turbo caching.